egs/swbd/s5c/local/segmentation/tuning/train_lstm_asr_sad_1a.sh

```diff
@@ -121,7 +121,7 @@ if [ $stage -le 6 ]; then
     --trainer.deriv-truncate-margin=10 \
     --trainer.max-param-change=$max_param_change \
     --trainer.compute-per-dim-accuracy=true \
-    --cmd="$decode_cmd" --nj 40 \
+    --cmd="$train_cmd" --egs.cmd="$train_cmd" --nj 40 \
```

Comment:
@vimalmanohar, can you check whether it's necessary to set egs.cmd here?  I would have thought it would default to 'cmd'.

Reply:
I am getting errors when creating egs on non-GridEngine env.

Reply:
https://github.com/kaldi-asr/kaldi/blob/018e79edbf4b1cf6f6b0ec7d5226808bf7b48be7/egs/wsj/s5/steps/nnet3/train_raw_rnn.py#L207
I defaults to cmd. So it should not be required.

Reply:
Actually, the default for egs_command should be changed to None
in https://github.com/kaldi-asr/kaldi/blob/018e79edbf4b1cf6f6b0ec7d5226808bf7b48be7/egs/wsj/s5/steps/libs/nnet3/train/common.py#L893

Reply:
yeah. Changing it to None would work.